css/dashboards/gui.css

```diff
@@ -749,28 +746,59 @@
     opacity: 0.3;
 }
 
-.highcharts-popup {
+/* Confirmation popup */
+
+.highcharts-dashboards-edit-popup {
     min-height: 160px;
     width: 200px;
     left: calc(50% - 150px);
+    display: none;
+    background-color: var(--highcharts-dashboards-main-background-color);
+    color: var(--highcharts-neutral-color-60);
+    font-size: 0.876em;
+    max-height: 90%;
+    top: 5%;
+    overflow-x: hidden;
+    overflow-y: hidden;
+    min-width: 300px;
+    max-width: 600px;
+    position: absolute;
+    z-index: 100;
+    -webkit-box-shadow: 0 0 8px 0 rgba(61, 61, 61, 0.3);
+    -moz-box-shadow: 0 0 8px 0 rgba(61, 61, 61, 0.3);
+    box-shadow: 0 0 8px 0 rgba(61, 61, 61, 0.3);
 }
 
-.highcharts-popup .highcharts-dashboards-edit-popup-content {
+.highcharts-dashboards-edit-popup-close {
+    background-repeat: no-repeat;
+    background-position: 50% 50%;
+    width: 20px;
+    height: 20px;
+    cursor: pointer;
+    position: absolute;
+    padding: 10px;
```

Comment:
In case of background-position 50/50%, you can declare only width/height. The padding is unecessary.